### PR TITLE
DVCI-123: Add FHIR instructions

### PR DESCRIPTION
**Pull request by Jammjammjamm**
This branch adds some fairly basic instructions for using the `$health-cards-issue` operation.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -22,13 +22,14 @@
   end
 
   post '/Patient/:patient_id/$health-cards-issue', to: 'health_cards#create', as: :issue_vc, format: :fhir_json
-  get "/Patient/:id", to: "patients#show", as: :fhir_patient, format: :fhir_json
-  get "/Immunization/:id", to: "immunizations#show", as: :fhir_immunization, format: :fhir_json
-  get "/.well-known/smart-configuration", to: "well_known#smart", as: :well_known_smart, format: :json
-  get "/.well-known/jwks", to: "well_known#jwks", as: :well_known_jwks, format: :json
-  get "/metadata", to: "metadata#capability_statement", as: :fhir_capabilitystatement, format: :fhir_json
-  get "/OperationDefinition/health-cards-issue", to: "metadata#operation_definition", as: :fhir_operationdefinition, format: :fhir_json
+  get '/Patient/:id', to: 'patients#show', as: :fhir_patient, format: :fhir_json
```
Comment (ssayer) [inline config/routes.rb:25]: I was curious why this wasn't caught by rubocop, and it looks like we're excluding the entire config directory. Should we re-enable that directory?

File: app/views/landing_page/fhir_instructions.html.erb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  }]
+}
+        </pre>
+      </p>
+      <p class="title">SMART on FHIR</p>
+      <p>
+        This server supports, but does not require, the
+        <%= link_to 'SMART on FHIR standalone app launch', 'http://hl7.org/fhir/smart-app-launch/index.html' %>
+        for granting access to the <code>$health-cards-issue</code> operation.
+        The standalone app workflow supports public apps using
```
Comment (ssayer) [inline app/views/landing_page/fhir_instructions.html.erb:47]: Do we need a caveat that this OAuth2 implementation isn't production worthy?

File: app/views/landing_page/fhir_instructions.html.erb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <p>
+        This server supports the
+        <%= link_to 'https://spec.smarthealth.cards/#via-fhir-health-cards-issue-operation' do %>
+          FHIR <code>$health-cards-issue</code> operation.
+        <% end %>
+      </p>
+      <p>
+        To retrieve a health card via this FHIR operation, POST a
+        <%= link_to 'Parameters resource', 'https://www.hl7.org/fhir/parameters.html' %> to
+        <code><%= root_url %>Patient/PATIENT_ID/$health-cards-issue</code>
```
Comment (ssayer) [inline app/views/landing_page/fhir_instructions.html.erb:25]: Just to make this idiot proof, add something saying to replace PATIENT_ID with the id of the patient?

File: config/routes.rb (hunk 1 above)
Comment (Jammjammjamm) [reply config/routes.rb:25]: I wouldn't because the majority of those files are generated.

**Review by ssayer — APPROVED** [review]

**Review by radamson — APPROVED** [review]

**Pull request merged**